src/parser/core/modules/Procs.tsx

```diff
@@ -479,76 +506,19 @@ export abstract class Procs extends Analyser {
 		this.addJobSpecificSuggestions()
 	}
 
-	private stopAndSave(procGroup: ProcGroup, event?: Event, type?: ProcStatusEventType): void {
-
-		// Only count usages if the event happens during uptime, you don't get credit for using an AoE-around-self Proc during downtime
-		if (event && type === 'usage' && !this.downtime.isDowntime(event.timestamp)) {
-			this.tryAddEventToUsages(procGroup, event)
-
-			// If the target of the cast was invulnerable, keep track of that too
-			if (this.invulnerability.isActive({
-				timestamp: event.timestamp,
-				actorFilter: actor => actor.id === (event as Events['action']).target,
-				types: ['invulnerable'],
-			})) {
-				this.tryAddEventToInvulns(procGroup, event)
-			}
-		}
-
-		// Count removals that occur during uptime, and the player is still alive. Don't need to double-penalize deaths or not having GCD space to use all your procs before a downtime
-		if (event && type === 'removal' && !(this.downtime.isDowntime(event.timestamp) || this.actors.current.hp.current === 0)) {
-			this.tryAddEventToRemovals(procGroup, event)
-		}
-
-		// If this was an overwrite event, and overwrites are disallowed for this proc, save a record of that
-		if (event && type === 'overwrite' && !procGroup.mayOverwrite) {
-			this.tryAddEventToOverwrites(procGroup, event)
-		}
-
-		if (!this.currentWindows.has(procGroup)) { return }
-
+	private stopAndSave(procGroup: ProcGroup, event?: Event, isOverwrite?: boolean): void {
```

Comment:
even if only two possible types, would rather keep the `type` - booleans in function args are really obtuse at the callsite.